### DISCUSSION TITLE
Unable to transfer domain out

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#6538] [Help]
### Summary

I've been trying to transfer my out domain from Vercel to others, but have been stuck with the same “Incorrect Authorization Code” message. Is there any way to reset the auth code?

the domain is inote.xyz

### Steps to Reproduce

Steps to Reproduce
Select my domain.
Press the transfer out button.
Copy the auth code.
Paste the code in the registrar's transfer form.
Receive “Incorrect Authorization Code” error message after a few seconds.

**Comment by ghost** [comment]
@amyegan please help me

**Reply by amyegan** [reply]
Your auth code is reset. You can get the new code from your Vercel dashboard.
